pipeline/download_gene.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2021, Blue Brain Project, EPFL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script that download and postprocess ISH dataset from Allen Brain."""
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+from pathlib import Path
+
+import numpy as np
+import PIL
+
+
+logger = logging.getLogger("download-gene")
+
+
+def parse_args():
+    """Parse command line arguments.
+
+    Returns
+    -------
+    args : argparse.Namespace
+        The parsed command line arguments.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--gene-name",
```

Comment:
So the idea is that we will get all experiments for a given `gene_name` and download them all, right? 

Would it not be realistic to assume that the user already found the experiment id and provide it right away?

Reply:
I would say (from my understanding) that the end goal is to have a complete volume for a given gene expression. For now, we do not combine the different experiments (in the interpolation step) but I think this should be done in the future. However, we can maybe ask Csaba and Dimitri's opinion tomorrow about this!

Reply:
let's discuss with them. IMO it adds a bit of ambiguity to the process but as you pointed out it might be useful in the future